.drone.yml

```diff
@@ -8,7 +9,12 @@ pipeline:
       - CGO_ENABLED=0
     commands:
       - go vet
-      - go test -cover -coverprofile=coverage.out
+      - |
+        for line in $(go list ./... | grep -v /vendor/)
+        do  
+          name=$(basename "$line")
+          go test "$line" -cover -coverprofile=coverage-"$name".out
```

Comment:
Do it like https://github.com/drone-plugins/drone-coverage/blob/release/0.4/Makefile#L25